build.ps1

```diff
@@ -181,21 +191,31 @@ function Compile {
 
 function GenerateOpenAPI {
     Invoke-Execute {
-        cd $solutionRoot/EdFi.Ods.AdminApi/
+        Push-Location $solutionRoot/EdFi.Ods.AdminApi/
         $outputOpenAPI = "../../docs/api-specifications/openapi-yaml/admin-api-$APIVersion.yaml"
         $dllPath = "./bin/Release/net8.0/EdFi.Ods.AdminApi.dll"
-        dotnet tool run swagger tofile --output $outputOpenAPI --yaml $dllPath v2
-        cd ..\..
+
+        try {
+            dotnet tool run swagger tofile --output $outputOpenAPI --yaml $dllPath v2
+        }
+        finally {
+            Pop-Location
+        }
```

Comment:
Sometimes a command fails after you've changed directories, and you don't notice that you're in a different directory now. That is frustrating. I always like to use `push` and then apply `pop` in a `finally` statement, helping ensure that I am back to the original directory when the script finishes executing.